lxd/operations.go

```diff
@@ -44,6 +47,66 @@ var operationWebsocket = APIEndpoint{
 	Get: APIEndpointAction{Handler: operationWebsocketGet, AllowUntrusted: true},
 }
 
+func waitForOperations(s *state.State) {
+	timeout := time.After(10 * time.Minute)
```

Comment:
We should reduce that to 8 minutes, the reason being that systemd will timeout at 10min and the snap packaging at 9min, so we should be done exiting prior to that.

Reply:
I suspect we'll probably want to make this configurable at some point, but for now, 8min should do :)

Reply:
Hmm, something I didn't consider is that instance shutdown itself can take a while as instances can have a shutdown delay... Given default timeout at systemd/snapd layer, we may want to try to go down to 5min and only raise it if we see people complaining.

Reply:
Can we intiate the shutdown wait concurrently with shutting down instances?